### ISSUE TITLE
New Enhanced behaviour for Video and Audio Roles in Final Cut Pro 10.7

### ISSUE THREAD

**Issue by IAmVigneswaran** [TheAcharya/MarkersExtractor#73]
Apple have just release Final Cut Pro 10.7.

- In 10.7 Users can assign custom Video roles to clips that has both Video and Audio.

https://www.youtube.com/watch?v=lC5qQWyTuZ0

Do we need to update our parsing logic for this?

**Comment by orchetect** [comment]
From what I can tell, it looks like a GUI feature of FCP and likely doesn't involve any changes with FCPXML data.

**Comment by latenitefilms** [comment]
Yes, this is my understanding too - it's just a visual change in the UI.

**Comment by IAmVigneswaran** [comment]
I also realise Apple has not updated the FCPXML version. It is still at 1.11.
